libs/components/packages/src/schematics/migrations/prettier-ignore-updates/prettier-ignore-updates.schematic.ts

```diff
@@ -0,0 +1,30 @@
+import { Rule } from '@angular-devkit/schematics';
+
+export default function addNewItemsToPrettierIgnore(): Rule {
+  return (tree) => {
+    const pathsToIgnore = [
+      '/src/assets',
+      '/projects/*/src/assets',
+      '/src/app/lib',
+    ];
+    const prettierIgnorePath = '/.prettierignore';
+    const prettierIgnore = tree.read(prettierIgnorePath);
+
+    let replacementString = prettierIgnore.toString();
+
+    for (const path of pathsToIgnore) {
+      if (
+        prettierIgnore &&
+        !new RegExp(`^` + path.replace('*', '\\*') + `$`, 'm').exec(
```

Comment:
The `.gitignore` patterns may include other characters that need escaping; to make this future-proof, you can copy an escape function I use in the Builders for this purpose; maybe put it in a new `/utility/regex.ts` file? See: https://dev.azure.com/blackbaud/Products/_git/skyux-angular-builders?path=/src/tools/webpack/plugins/app-assets/app-assets.plugin.ts&version=GB5.x.x&line=17&lineEnd=24&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents

Reply:
Good call. Done